infra-as-code/bicep/modules/vnetPeering/README.md

```diff
@@ -1,4 +1,4 @@
-# Module: Virtual Network Peer
+# Module: VNet Peering
 
 This module creates a virtual network peer between two virtual networks and is to be utilized by other modules.  Module will need to be called twice to create the completed peering.  Each time with a peering direction. This allows for peering between different subscriptions.  
```

Comment:
Can we update the first sentence to talk about virtual network peering instead of peer?

Reply:
grammatical change has been made on latest commit [39e476a]